pkg/skaffold/util/http.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2019 The Skaffold Authors
```

Comment:
```suggestion
Copyright 2020 The Skaffold Authors
```

Reply:
This code was copied out and modified, so it really is 2019.